docs/zh-CN/docs/sql-manual/sql-functions/array-functions/array_last.md

```diff
@@ -0,0 +1,70 @@
+---
+{
+    "title": "array_last",
+    "language": "zh-CN"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## array_last
+
+<version since="2.0">
+
+array_last
+
+</version>
+
+### description
+Returns the last element in the array for which func(arr1[i]) returns something other than 0.
```

Comment:
this line maybe could remove in zh-CN docs

Reply:
done